src/mysql/SchemaFactory.cc

```diff
@@ -107,6 +107,16 @@ struct ColTypeFactory {
         sqlType = os.str();
     }
 
+    void _setDateTime(char const* baseType, int length) {
+        if (length == 0) {
+            sqlType = baseType;
+        } else {
+            std::ostringstream os;
+            os << baseType << "(" << length << ")";
+            sqlType = os.str();
```

Comment:
C++ streams are rather expensive, you could probably make it more efficient with the code like:

    sqlType = baseType;
    sqlType += '(';
    sqlType += std::to_string(length);
    sqlType += ')';

Reply:
Fixed